code/obj/item/storage/med_chem.dm

```diff
@@ -30,14 +30,14 @@
 	/obj/item/reagent_containers/emergency_injector/epinephrine)
 
 	// Comes with upgraded health scanner.
-	doctor_spawn
-		spawn_contents = list(/obj/item/reagent_containers/mender/brute,\
-		/obj/item/reagent_containers/mender/burn,\
-		/obj/item/reagent_containers/patch/bruise,\
-		/obj/item/reagent_containers/patch/burn,\
-		/obj/item/reagent_containers/pill/salicylic_acid,\
-		/obj/item/device/analyzer/healthanalyzer/upgraded,\
-		/obj/item/reagent_containers/emergency_injector/epinephrine)
+	prepared
+		spawn_contents = list(/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/device/analyzer/healthanalyzer/upgraded,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/glass/beaker/antitox,
+		/obj/item/reagent_containers/glass/beaker/epinephrine,
+		/obj/item/reagent_containers/glass/beaker/salbutamol)
```

Comment:
hypos and beakers do not belong in first aid kits.

Reply:
Swapped with large auto injectors (know its a nukie thing however the aid kit will also be an admin thing so, tell me if bad)